lib/class-wp-theme-json-resolver-gutenberg.php

```diff
@@ -563,6 +563,21 @@ public static function get_user_data() {
 			}
 		}
 
+		/*
+		 * The Separator block uses different CSS properties for its color depending
+		 * on how it is being rendered e.g. as "content" for the Dots style, or
+		 * as a border etc.
+		 * 
+		 * Uses are only presented with a single color control for background. Any
+		 * selection of a background color should be applied to the other paths
+		 * so it can be honoured.
+		 */
+		$separator_color =  $config['styles']['blocks']['core/separator']['color']['background'] ?? null;
```

Comment:
This reminds me to brush off the [theme json style-related methods abstraction issue](https://github.com/WordPress/gutenberg/issues/65728)